### ISSUE TITLE
[Request]  Add aesthetic-rating complimented dataset support into the discriminator loss function

### ISSUE THREAD

**Issue by nom57** [NVlabs/stylegan2-ada-pytorch#256]
Add aesthetic-rating aided dataset support into the discriminator loss function.
Add a function that could penalize the discriminator for thinking an image from the dataset is "more real" than one which has a higher aesthetic-score rating (could be an aesthetic score system beetwen 1 to 5 or 1 to 10).

This would greatly help use-cases where you have a lot of images for a dataset but only a select few are good enough in quality and creativity, but you still wanna keep the diversity of a much larger dataset.
